common/djangoapps/student/views/login.py

```diff
@@ -465,6 +465,26 @@ def login_user(request):
                 # Important: This call must be made AFTER the user was successfully authenticated.
                 _enforce_password_policy_compliance(request, possibly_authenticated_user)
 
+        # Check whether both users (student and parent) are active
+        if possibly_authenticated_user and possibly_authenticated_user.is_active:
+            StudentProfile = apps.get_model('tedix_ro', 'StudentProfile')
+            ParentProfile = apps.get_model('tedix_ro', 'ParentProfile')
+            try:
+                # parent is logging in
+                if possibly_authenticated_user.parentprofile:
```

Comment:
Why you use `if` here? You will get an exception for non-exists parent profile.

Reply:
removed